### ISSUE TITLE
Animation Pause

### ISSUE THREAD

**Issue by sw1ftc0d3r** [passsy/android-HoloCircularProgressBar#14]
Hi, Can I pause and after re-start the animation ?

**Comment by passsy** [comment]
Yes. Look at the sample application. I use the [ValueAnimator](http://developer.android.com/reference/android/animation/ValueAnimator.html) to animate the progress bar. And the ValueAnimator supports `pause()` and `resume()`.

**Comment by sw1ftc0d3r** [comment]
Thank's for your reply, pause() and resume() is added on API 19.

**Comment by GrapeJelly** [comment]
What's the easiest way to do this on earlier APIs?

**Comment by isheeley** [comment]
I would also like to know how to do this on earlier APIs

**Comment by mbleftley** [comment]
Anything new on this problem of using pause() on earlier APIs?
